operator/internal/central/extensions/reconcile_pvc.go

```diff
@@ -36,72 +39,133 @@ const (
 	// PVCTargetCentral is for any PVC that would be attached to the Central deployment
 	PVCTargetCentral PVCTarget = "central"
 
-	// PVCTargetCentralDB is for any PVC that would be attached to the Central DB deployment
+	// PVCTargetCentralDB is for a PVC that would be attached to the Central DB
+	// deployment as a data volume
 	PVCTargetCentralDB PVCTarget = "central-db"
+
+	// PVCTargetCentralDBBackup is for a PVC that would be attached to the
+	// Central DB deployment as a backup volume
+	PVCTargetCentralDBBackup PVCTarget = "central-db-backup"
 )
 
 var (
 	errMultipleOwnedPVCs = errors.New("operator is only allowed to have 1 owned PVC")
 
-	defaultPVCSize = resource.MustParse("100Gi")
+	DefaultPVCSize       = resource.MustParse("100Gi")
+	DefaultBackupPVCSize = resource.MustParse("200Gi")
```

Comment:
Would
```suggestion
	DefaultBackupPVCSize = DefaultPVCSize*2
```
work and make the connection explicit?

Reply:
Quantity struct doesn't support such simple arithmetic operation. In the end it has to be a deep copy + making it a pointer and call Mul on it. All those manipulation distract heavily from the main idea, so I've suggest to keep it as it is.